### PR TITLE
PulsePoint ads - update for deprecation of delayed fetch in doubleclick.js (#11834)

### PR DESCRIPTION
**Pull request by anand-venkatraman**
PulsePoint ads update for deprecation of delayed fetch in doubleclick.js.

`fixes` pulsepoint ads dependency on doubleclick.js. https://github.com/ampproject/amphtml/issues/11834

### CODE REVIEW

**Comment by anand-venkatraman** [comment]
I signed it!

**Comment by lannka** [comment]
@anand-venkatraman could you please resolve the confict

File: ads/pulsepoint.js [hunk 1]
```diff
@@ -14,23 +14,17 @@
...
 
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
 export function pulsepoint(global, data) {
-  // TODO: check mandatory fields
   validateData(data, [], [
-    'pid', 'tagid', 'tagtype', 'slot', 'timeout',
+    'pid', 'tagid',
```
Comment (lannka) [inline ads/pulsepoint.js]: FYI, this is where you specifying optional fields. Just to confirm this is what you want.

**Comment by nainar** [comment]
Pinging @anand-venkatraman for the next action.

File: ads/pulsepoint.js (hunk 1 above)
Comment (anand-venkatraman) [reply ads/pulsepoint.js]: Thanks for pointing it out, fixed it.

**Comment by anand-venkatraman** [comment]
Addressed merge issue and fixed comment (sorry for the delay with this). Travis CI is failing, but I think it is for an unrelated reason. Thanks!

File: validator/testdata/feature_tests/ads.out [hunk 2]
```diff
@@ -1042,17 +1042,7 @@ PASS
...
-|    <amp-ad width="300" height="250"
-|        type="pulsepoint"
-|        data-pid="521732"
-|        data-tagid="76835"
-|        data-tagtype="hb"
-|        data-timeout="1000"
-|        data-slot="/1066621/ExchangeTech_Prebid_AdUnit">
-|    </amp-ad>
-|
+|
```
Comment (lannka) [inline validator/testdata/feature_tests/ads.out]: extra space? this is failing the integration test

**Comment by lannka** [comment]
hi @anand-venkatraman are you still on this? Let me know if you need any help.

File: validator/testdata/feature_tests/ads.out (hunk 2 above)
Comment (anand-venkatraman) [reply validator/testdata/feature_tests/ads.out]: Fixed this, but travis build is still failing, logs dont seem to indicate what the problem is, and running `node build-system/pr-check.js` on my local machine also seems to be fine. Pls let me know if there is anything else to fix. Thanks!

**Comment by lannka** [comment]
Closing for inactivity. feel free to reopen

**Pull request closed without merging**